context/app/static/js/components/workspaces/AdvancedConfigOptions/style.ts

```diff
@@ -0,0 +1,32 @@
+import { styled } from '@mui/material/styles';
+import Typography from '@mui/material/Typography';
+import Switch from '@mui/material/Switch';
+import Accordion from '@mui/material/Accordion';
+
+const StyledSubtitle = styled(Typography)(({ theme }) => ({
+  ...theme.typography.subtitle2,
+  color: theme.palette.text.primary,
+}));
+
+const StyledHeader = styled(StyledSubtitle)({
+  fontSize: '1rem',
+});
+
+const StyledSwitchLabel = styled(Typography)({
+  fontSize: '.75rem',
+});
+
+const StyledSwitch = styled(Switch)(({ theme }) => ({
```

Comment:
Can we use the `color` prop instead?

Reply:
It looks like this doesn't impact the thumb of the switch unfortunately.